manager/orchestrator/updater.go

```diff
@@ -224,10 +222,32 @@ func (u *Updater) updateTask(ctx context.Context, service *api.Service, original
 		case e := <-taskUpdates:
 			updated = e.(state.EventUpdateTask).Task
 			if updated.Status.State >= api.TaskStateRunning {
```

Comment:
It's related to #486. I think a single task failure shouldn't stop update. We should define a stop threshold for each service. Service update can abort, or rollback.